docs-kits/kits/behaviour-twin-kit/use-cases/rul/overview.md

```diff
@@ -0,0 +1,37 @@
+﻿---
+id: overview
+title: Remaining useful Life
+description: Behaviour Twin KIT
+---
+
+<div style={{fontSize:30, color:'rgb(255,166,1)', marginBottom:13}}>Behaviour Twin KIT -- Remaining useful Life</div>
```

Comment:
Now, I set hide_title: true.

HTML is used to provide color for that special headline. # would not work because it would be a multiple top level headline. By setting hide_title: true, I got rid of the redundant headline now.